src/models.py

```diff
@@ -221,24 +221,26 @@ def _get_categorical_indeces(self, feature_names, categorical_features):
             return None
 
 
-class SklearnTransformer(BaseTransformer):
+class CategoryEncoder(BaseTransformer):
```

Comment:
Honestly I really don't like the name, because there is 'CategoricalEncoder' in feature extraction and it's confusing.

IMO, it would be ideall to somehow leave OneHotEncoder step as it was and somehow transfer new categorical_features to sklearn_preprocess, but I know that might be complicated.

Reply:
@pknut I agree 100% with @kstrzala it should be called OneHotEncoder because that's what it is. Now we don't need to transfer anything from sklearn_preprocess because we have xgb_preprocess and those 2 are not used together right?

Reply:
I forgot about 'CategoricalEncoder' in feature extraction. You're right.

Reply:
Right, when seeing this I always think about sklearn, forgetting that it's used for xgboost.